scripts/db_connect.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# This script requires google's cloud_sql_proxy on your PATH. One way to set this up:
+# $ brew cask install google-cloud-sdk
+# $ gcloud components install cloud_sql_proxy
+# $ PATH=$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/
+# OR
+# $ ln -s /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/cloud_sql_proxy /usr/local/bin/cloud_sql_proxy
+#
+# Use this script to connect to cloud Postgres for a specific Catalog instance.
+# For example, to connect to the dev catalog database, run:
+# $ ENV=dev ./db-connect.sh
+#
+# The proxy will continue to run until you quit it using ^C.
+#
+# The default port used is 5431, to avoid conflicting with a locally running postgres which
+# defaults to port 5432. Use the environment variable PORT to override this setting.
+
+: "${ENV:?}"
+PORT=${PORT:-5431}
+
+VAULT_PATH=secret/dsde/terra/kernel/${ENV}/${ENV}/catalog/postgres
```

Comment:
```suggestion
VAULT_PATH="secret/dsde/terra/kernel/${ENV}/${ENV}/catalog/postgres"
```

I recommend quoting all strings for clarity, consistency with POSIX sh, and potential wildcard expansion. Has this script been checked with shellcheck?

Reply:
I can add the quotes back, but they have no effect in this case. Shellcheck doesn't flag this as an issue.